### PR TITLE
Erick week2

### PR DESCRIPTION
**Pull request by ert224**
Week_2 Spam components

### CODE REVIEW

**Comment by ert224** [comment]
Pull request submitted for week 2

File: public/css/erick-week2.css [hunk 1]
```diff
@@ -10,5 +11,211 @@ html {
     padding-left: 9rem; 
     padding-right: 9rem;
     margin: 0 auto;
+}
+
+#AmazonSeg{
```
Comment (jossTripoli) [inline public/css/erick-week2.css:16]: use kebab case like: amazon-seg

File: public/css/erick-week2.css [hunk 2]
```diff
@@ -10,5 +11,211 @@ html {
...
+    border: 1px solid #9D9D9D;
+    background: #FFF;
     
-}
\ No newline at end of file
+    display: flex;
+    width: 630px;
+    height: 600px;
+    padding: 40px;
+    flex-direction: column;
```
Comment (jossTripoli) [inline public/css/erick-week2.css:25]: you can do the positioning with formantic ui instead of all this css

File: public/css/erick-week2.css [hunk 3]
```diff
@@ -10,5 +11,211 @@ html {
...
+    align-items: center;
+    gap: 21px;
+}
+
+#AmazonStack {
+    width: 602px;
```
Comment (jossTripoli) [inline public/css/erick-week2.css:31]: Avoid using static css units like pixel. To be responsive we want to base the size in relation to the container

File: views/people/erick/week2.pug [hunk 4]
```diff
@@ -11,12 +11,95 @@ block content
...
-
+            br
+            
+            .ui.segment#AmazonSeg
+                .item#item1
+                    img.AmazonLogo(src="/images/erick/week2/Amazon_logo.svg")
+                    h3 Password Assistance
+                .item#item2
+                    .ui.stackable.vertical.grid#AmazonStack
+                        .row#rowAmazon
```
Comment (jossTripoli) [inline views/people/erick/week2.pug:22]: IDs can't be repeated for more than one element, they can only be made unique to one. Class should be used to style multiple features or just style the element themself but in relation to its parent to limit its scope

File: views/people/erick/week2.pug [hunk 5]
```diff
@@ -11,12 +11,95 @@ block content
...
+                            p We hope to see you again soon.
             h1 Dropbox 
             //- code here
+            .ui.segment#DropboxSeg
+                    .ui.stackable.vertical.grid#DropStackable
+                        .row
+                            img.DropboxLogo(src="/images/erick/week2/Dropbox_logo.svg")
+                        .row
+                            span.spanDroptxt We noticed you logged into dropbox using
```
Comment (jossTripoli) [inline views/people/erick/week2.pug:37]: Want to try to rely on the formantic ui styling so use h1,h2,p, etc instead of spans with custom styling

**Review by jossTripoli — APPROVED** [review]

**Pull request merged**